### PR TITLE
fix some typos / mistakes in README

### PR DESCRIPTION
**Pull request by jxdv**
^title

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,18 +32,18 @@
...
 
 reNgine 2.0-jasper is out now, you can [watch reNgine 2.0-jasper release trailer here!](https://youtu.be/VwkOWqiWW5g)
 
-reNgine 2.0-Jasper would not have been possible without [@ocervell](https://github.com/ocervell) valuable contributions. [@ocervell](https://github.com/ocervell) did majority of the refactoring if not all and also added a ton of features. Together, we wish to shape the future of web application reconnaissance, and it's developers like [@ocervell](https://github.com/ocervell) and a [ton of other developers and hackers from our community](https://github.com/yogeshojha/rengine/graphs/contributors) who inspire and drive us forward.
+reNgine 2.0-Jasper would not have been possible without [@ocervell](https://github.com/ocervell) valuable contributions. [@ocervell](https://github.com/ocervell) did the majority of the refactoring if not all and also added a ton of features. Together, we wish to shape the future of web application reconnaissance, and it's developers like [@ocervell](https://github.com/ocervell) and a [ton of other developers and hackers from our community](https://github.com/yogeshojha/rengine/graphs/contributors) who inspire and drive us forward.
```
Comment (AnonymousWP) [inline README.md:42]:
```suggestion
reNgine 2.0-Jasper would not have been possible without [@ocervell](https://github.com/ocervell) valuable contributions. [@ocervell](https://github.com/ocervell) did the majority of the refactoring (if not all) and also added a ton of features. Together, we wish to shape the future of web application reconnaissance, and it's developers like [@ocervell](https://github.com/ocervell) and a [ton of other developers and hackers from our community](https://github.com/yogeshojha/rengine/graphs/contributors) who inspire and drive us forward.
```

**Review by AnonymousWP — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -457,14 +457,14 @@ Security researchers, welcome aboard! I'm excited to announce the reNgine bug bo
...
-**Please do not disclose vulnerabilities via Github issues/blogs/tweets after/before reporting to huntr.dev as this is explicitly against the disclosure policy of huntr.dev and reNgine and will not be considered for monetary rewards.**
+**Please do not disclose vulnerabilities via GitHub issues/blogs/tweets after/before reporting to huntr.dev as this is explicitly against the disclosure policy of huntr.dev and reNgine and will not be considered for monetary rewards.**
 
 Please note that the reNgine maintainer does not set the bounty amount.
 The bounty reward is determined by an industry-first equation developed by huntr.dev to understand the popularity, impact and value of repositories to the open-source community.
 
 **What do I expect from security researchers?**
 
-* Patience: Please note that I am currently the only maintainer in reNgine and it will take some time to validate your report. I ask for your patience during this process.
+* Patience: Please note that I am currently the only maintainer in reNgine, and it will take some time to validate your report. I ask for your patience during this process.
```
Comment (AnonymousWP) [inline README.md:467]:
```suggestion
```

Can be removed as we're with 3 maintainers now.

**Review by AnonymousWP — APPROVED** [review]

**Review by AnonymousWP — CHANGES_REQUESTED** [review]
Oops, I meant to request, not approve.

**Review by AnonymousWP — APPROVED** [review]

**Comment by yogeshojha** [comment]
@jxdv please fix merge conflict.

**Comment by jxdv** [comment]
> @jxdv please fix merge conflict.

Done.

**Review by psyray — APPROVED** [review]

**Pull request merged**